typescript/infra/config/environments/mainnet3/warp/bsc-lumia-LUMIA-deployments.yaml

```diff
@@ -10,6 +10,7 @@ data:
       type: collateral
       hypAddress: '0xdD313D475f8A9d81CBE2eA953a357f52e10BA357'
       tokenAddress: '0xd9343a049d5dbd89cd19dc6bca8c48fb3a0a42a7'
```

Comment:
I'm getting a little confused with this setup, theres a Lumia ERC20 on Ethereum and also Lumia tokens on Lumia L2, does that make sense? Do they have a canonical bridge?

Reply:
would chat to @cmcewen

Reply:
unfortunately I don't have any context here, agree Connor is best

Reply:
Will look into this as a follow up